cnvlib/commands.py

```diff
@@ -1006,6 +1007,10 @@ def _cmd_scatter(args):
         help="Draw a smoothed local trendline on the scatter plot.")
 P_scatter_aes.add_argument('--y-max', type=float, help="y-axis upper limit.")
 P_scatter_aes.add_argument('--y-min', type=float, help="y-axis lower limit.")
+P_scatter_aes.add_argument('-p', '--plot-dim', nargs=2, type=int,
```

Comment:
```suggestion
P_scatter_aes.add_argument('-p', '--plot-dim', nargs=2, type=float,
```

Since dimentions can be float in MPL